endpoints/api_config.py

```diff
@@ -333,11 +337,17 @@ def audiences(self):
     return self.__common_info.audiences
 
   @property
-  def scopes(self):
+  def scope_objs(self):
```

Comment:
Maybe adjust the docstring for this one too?
If I understand correctly:
 - `scope_objs` reports a list of OAuth2Scope instances
 - `scopes` reports a list of OAuth scope (strings)